pipeline_reviews/transform.py

```diff
@@ -0,0 +1,92 @@
+"""Validates received review inputs"""
+
+from datetime import datetime
+from os import environ
+
+import pandas as pd
+from pandas import DataFrame
+from dotenv import load_dotenv
+from psycopg2.extras import RealDictCursor
+from psycopg2 import connect
+from psycopg2.extensions import connection
+
+
+def get_db_connection() -> connection:
+    """Returns PSQL database connection"""
+    load_dotenv()
+    return connect(dbname=environ["DATABASE_NAME"],
+                    user=environ["DATABASE_USERNAME"],
+                    password=environ["DATABASE_PASSWORD"],
+                    cursor_factory=RealDictCursor)
+
+
+def get_release_date(game_id: int, conn: connection, cache: dict) -> datetime:
+    """Retrieves the release date for a game with a provided ID"""
+    if game_id in cache.keys():
+        return cache[game_id]
+    else:
+        # TODO get data from connection
```

Comment:
is this a todo for you/ is this already done

Reply:
It is. I just was waiting for data I could use from the database